src/Sylius/Bundle/ApiBundle/Serializer/CartShippingMethodSerializer.php

```diff
@@ -71,17 +71,20 @@ public function getSubresource(string $resourceClass, array $identifiers, array
 
         Assert::true($cart->hasShipment($shipment), 'Shipment doesn\'t match for order');
 
+        $shippingMethod = $shipment->getMethod();
+        $calculator = $this->calculators->get($shippingMethod->getCalculator());
+
+        /** @var int $price */
+        $data['price'] = $calculator->calculate($shipment, $shippingMethod->getConfiguration());
```

Comment:
Are those lines actually needed? `$data` is an array not-passed by reference so it should have no effect (and the price comes from `getCartShippingMethods` in this case).